lib/web/apiserver.go

```diff
@@ -920,7 +920,10 @@ func (h *Handler) bindDefaultEndpoints() {
 
 	// MFA private endpoints.
 	h.GET("/webapi/mfa/devices", h.WithAuth(h.getMFADevicesHandle))
+	h.DELETE("/webapi/mfa/devices", h.WithAuth(h.deleteMFADeviceHandle))
 	h.POST("/webapi/mfa/authenticatechallenge", h.WithAuth(h.createAuthenticateChallengeHandle))
+	h.POST("/webapi/mfa/registerchallenge", h.WithAuth(h.createRegisterChallengeHandle))
```

Comment:
Does the UI code degrade gracefully if it gets a 404 for either of these new endpoints?

Reply:
They aren't in use yet - 3b46013e069419f867b7361eb4681668d10e2158